playwright.config.ts

```diff
@@ -0,0 +1,90 @@
+import { PlaywrightTestConfig } from '@playwright/test'
+import { devices } from '@playwright/test'
+import os from 'os'
+
+const projects = [
+  {
+    name: 'chromium',
+    use: {
+      ...devices['Desktop Chrome'],
+      permissions: ['clipboard-read', 'clipboard-write'],
+      launchOptions: {
+        // headless: false,
+        /**
+         * Enable scrollbars in headless mode.
+         */
+        ignoreDefaultArgs: ['--hide-scrollbars'],
+      },
+    },
+  },
+  {
+    name: 'firefox',
+    use: {
+      ...devices['Desktop Firefox'],
+    },
+  },
+]
+
+if (os.type() === 'Darwin') {
+  projects.push({
+    name: 'webkit',
+    use: {
+      ...devices['Desktop Safari'],
+    },
+  })
+}
+
+const retries = process.env.PLAYWRIGHT_RETRIES
+  ? +process.env.PLAYWRIGHT_RETRIES
+  : process.env.CI
+  ? 5
```

Comment:
5 retries in CI. If any test fails, it gets rerun and you get a flaky test report at the end of the run.

https://github.com/ianstormtaylor/slate/actions/runs/3859077663/jobs/6583635166#step:4:291